libr/core/disasm.c

```diff
@@ -1512,13 +1512,6 @@ static int handleMidFlags(RCore *core, RDisasmState *ds, bool print) {
 			} else if (!strncmp (fi->name, "str.", 4)) {
 				ds->midflags = R_MIDFLAGS_REALIGN;
 			} else if (!strncmp (fi->name, "reloc.", 6)) {
-				if (print) {
```

Comment:
Not only it seems unnecessary because, as written in the comment, there is already a comment for relocations. But it is also wrong in some cases.